support-frontend/assets/components/asyncronously/asyncronously.ts

```diff
@@ -1,47 +1,38 @@
-import type { ComponentType, Node } from 'react';
-import { Component } from 'react';
+import type { ComponentType } from 'preact';
```

Comment:
We don't generally import from `preact`, is there a reason why we need to here?

Reply:
No there's no particular reason to do so here. Will revert this change but wondering for future reference why we might not want to import from preact?

Reply:
Ah yeah that makes sense, though you lost me a little bit at the end there what did you mean by
> ... I suppose it would be possible to drop `preact` and just `react` by just removing the alias.